utam-tests/src/test/java/utam/base/UtamMobileTestBase.java

```diff
@@ -70,7 +70,7 @@ protected final void setupIOS() {
     setupMobile();
     System.setProperty("ios.device", "iPhone 8 Plus");
     System.setProperty("ios.app", getUserHomeRelativePath("SApp.app"));
-    desiredCapabilities.setDesiredCapability(MobileCapabilityType.PLATFORM_VERSION, "13.4");
+    desiredCapabilities.setDesiredCapability(MobileCapabilityType.PLATFORM_VERSION, "15.2");
```

Comment:
the published debug app version has move the minimum support ios version from 13.4 to 14.3 now.